cmd/pFaTools/extractBed.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fasta/pFasta"
+	"log"
+	"os"
+)
+
+// ExtractBedSettings defines the usage settings for the pFa extractBed subcommand.
+type ExtractBedSettings struct {
+	InFile  string
+	Region  string
+	OutFile string
+	TakeCoords bool
+}
+
+// extractBedUsage defines the usage statement for the pFaTools extractBed subcommand.
+func extractBedUsage(extractBedFlags *flag.FlagSet) {
+	fmt.Printf("pFaTools extractBed - a tool for extracting a subsequence from a pFa, as specified by a Bed file.\n" +
```

Comment:
Can we update this usage to reflect the fact that now we can extract multiple regions at once?

Reply:
Added!